samba-integration-centos-ci-tests.sh

```diff
@@ -73,8 +74,9 @@ scl enable sclo-vagrant1 -- \
 
 # time to run the tests:
 
+make "${TEST_TARGET}"
 # TODO: add real tests to execute
-true
+# When the tests use vagrant, run them like so:
 #echo make "${TEST_TARGET}" | scl enable sclo-vagrant1 bash
```

Comment:
You can just run it in a "vagrant enabled shell" already, nothing should prevent functioning.

Reply:
what does that mean 'vagrant-enabled shell' - how do I do that?

Reply:
Just use `echo make "${TEST_TARGET}" | scl enable sclo-vagrant1 bash` and be done with it. Even if you do not run any vagrant in that `scl ... bash` shell, things should just work.

Reply:
ah! ok :-)

Reply:
updated